lib/ui/form/widget/form_survey_answer_textarea.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
+
+class FormSurveyAnswerTextarea extends StatefulWidget {
```

Comment:
Is it possible that this widget can be a `StatelessWidget` instead of a `StatefulWidget` ?
```suggestion
class FormSurveyAnswerTextarea extends StatelessWidget {
```

Reply:
@Tuubz I think we need the `dispose()` function from `StateFulWidget` to dispose `TextEditingController` to avoid memory leak. Please correct me if I'm wrong though 😄